sotopia/cli/benchmark/benchmark.py

```diff
@@ -275,24 +324,67 @@ def benchmark(
     batch_size: int = typer.Option(10, help="The batch size you want to use."),
     task: str = typer.Option("hard", help="The task id you want to benchmark."),
     print_logs: bool = typer.Option(False, help="Print logs."),
+    only_show_performance: bool = typer.Option(False, help="Only show performance."),
```

Comment:
Maybe just skip the for-loop if `only_show_performance`? Since `only show performance` implies that performance will also be shown. And save to jsonl can also be an argument: the user can use the customized json, or False if they don't want it printed.